src/datasets/info.py

```diff
@@ -108,6 +109,7 @@ class DatasetInfo:
         post_processing_size (int, optional):
         dataset_size (int, optional):
         size_in_bytes (int, optional):
+        task_templates (List[TaskTemplate], optional): TODO(sbrandeis) - document this
```

Comment:
@lhoestq should document this attribute of the DatasetInfo?

Reply:
looking at the [docs](https://huggingface.co/docs/datasets/master/package_reference/main_classes.html?highlight=datasetinfo#datasetinfo) it seems that none of the `DatasetInfo` attributes are documented. perhaps this is best addressed in a separate PR?

Reply:
Fine by me :)

Reply:
i created an issue for this in #2354 and will tackle it on friday :)